defaults/main.yml

```diff
@@ -43,6 +43,13 @@ siteName: "io"
 nodeBase: "{{ siteName }}"
 nodeGpuBase: "gpu"
 
+# Have systemd restart slurm daemons if they fail to start
+slurm_systemd_override_slurmd: True
+slurm_systemd_override_slurmdbd: False
```

Comment:
I think this could default to True? If the daemon crashes, we're not losing anything by trying to restart it, do we?